packages/millicast-sdk/src/workers/TransformWorker.worker.js

```diff
@@ -1,44 +1,78 @@
 import { addH26xSEI, extractH26xMetadata } from '../utils/Codecs'
 
-let uuid = ''
-let payload = ''
+const DROPPED_SOURCE_TIMEOUT = 2000
+const metadata = []
 let codec = ''
 let codecMap = {}
+const synchronizationSources = {}
```

Comment:
Could you please add comment to explain the purpose of the synchronizationSources object?